src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1138,6 +1138,34 @@ public FileDialog GetSaveDialog(WorkspaceModel workspace)
         }
 
         /// <summary>
+        /// Attempts to open a file using the passed open command, but wraps the call
+        /// with a check to make sure no unsaved changes to the HomeWorkspace are lost.
+        /// </summary>
+        /// <param name="openCommand"> <see cref="DynamoModel.OpenFileCommand"/> </param>
+        private void OpenIfSaved(object openCommand)
+        {
```

Comment:
Is this command specific to Revit playlist ? If OpenFileCommand is a delegate, can't we just do a check there?
may be,  I am dumb asking this question